benchmarks/source-cosmicjs/package.json

```diff
@@ -13,6 +13,7 @@
     "serve": "gatsby serve"
```

Comment:
Can we add a data-update package.json script that invokes the `update-post.js` command? a la this: https://github.com/gatsbyjs/gatsby/pull/24517/files#diff-c11783c10707e9768f19927f7a98031a

Reply:
@DSchau I added